src/blog/2025/06/flowfuse-forms-easy-data-collection-factory-floor.md

```diff
@@ -0,0 +1,336 @@
+---
+title: "FlowFuse Forms: Easy Data Collection for Your Factory Floor"
+subtitle: Get Shop Floor Data Directly to Machine Settings
+description: Learn how to create and configure forms in FlowFuse Dashboard to collect data efficiently in industrial applications using Node-RED.
+date: 2025-06-09
+authors: ["sumit-shinde"]
+image: /blog/2025/06/images/building-forms-in-flowfuse.png
+keywords: flowfuse form, node-red form, Recipe Management System, Node-RED Recipe Management, FlowFuse Recipe Management System
+tags:
+   - flowfuse
+---
+
+It's often a pain to get important data from the factory floor. Things like updating recipes, doing quality checks, or setting up machines still rely on old methods – manual notes, slow spreadsheets. This can lead to delays, errors, and a lot of wasted time before anyone can actually use the information. It's especially tough when you need quick feedback from an operator or a specific setting to go straight to a machine.
+
+<!--more-->
+
+This post will show you a better way. We'll look at how FlowFuse Forms on the FlowFuse Dashboard can make collecting factory data much simpler. You'll learn how to build useful forms that connect what your team knows directly to your industrial processes. We'll even walk through a example for digitizing machine recipe updates, showing you how to build this solution yourself, step-by-step, which will include everything we learned so far about form.
+
+## Prerequisites
+
+Before you begin, make sure you have the following:
+
+- **Running FlowFuse Instance:** Make sure you have a FlowFuse instance set up and running. If you don't have an account, check out our [free trial](https://app.flowfuse.com/account/create).
```

Comment:
They're not running a "FlowFuse Instance" - that's the whole platform. They need a copy of Node-RED. 

```suggestion
- **Node-RED:** Make sure you have an instance of Node-RED up and running. The quickest way to do this is via. If you don't have an account, check out our [free trial](https://app.flowfuse.com/account/create).
```